app/main.py

```diff
@@ -29,10 +29,10 @@ def wash_single_car(self, car: Car):
         return car.clean_mark
 
     def serve_cars(self, list_of_cars: list):
-        sorted_list_of_cars = [car for car in list_of_cars
+        filtered_list_of_cars = [car for car in list_of_cars
                                if car.clean_mark < self.clean_power]
```

Comment:
You also need to shift this line by a few spaces so that flake8 test passes. :)

Reply:
All done✓